docs/source/intro.rst

```diff
@@ -0,0 +1,44 @@
+===========
+ome-zarr-py
+===========
+
+Experimental support for multi-resolution images stored in Zarr filesets, according to the `OME zarr spec`_.
```

Comment:
OME zarr spec link leads to a repo called ome/ngff - potential for confusion ? (no mention of ``zarr`` on that page)